### ISSUE TITLE
Register language icon

### ISSUE THREAD

**Issue by gitonthescene** [exercism/bqn#7]
I'm guessing this hasn't been done yet, but we can use this issue to coordinate steps taken.  One of the first steps listed in [preparing for launch](https://exercism.org/docs/building/tracks/new/prepare-for-launch) is to open an issue to register an icon.

I'm guessing this counts as the [official icon](https://github.com/mlochbaum/BQN/blob/d6abe6879f53763bb1283a7721676be9e7f6c27c/logo.svg).

If there's nothing posted the next time I check on this issue, I'll go ahead and start the process to register this icon.

**Comment by razetime** [comment]
yes, that's the official icon, and we should use that wherever possible.

**Comment by razetime** [comment]
An issue is now made: https://github.com/exercism/website-icons/issues/145
